src/lib/y2storage/storage_manager.rb

```diff
@@ -59,7 +59,25 @@ class StorageManager
     # @return [Proposal, nil]
     attr_reader :proposal
 
-    def_delegators :@storage, :environment, :arch
+    def_delegators :@storage, :environment, :arch, :rootprefix
+
+    # @!method rootprefix
+    #   @return [String] root prefix used by libstorage
+
+    # Sets the root prefix used by libstorage in subsequent operations
+    #
+    # @param path [String]
+    def rootprefix=(path)
+      storage.rootprefix = path
+    end
+
+    # Prepends the current libstorage root prefix to a mountpoint if necessary
+    #
+    # @param path [String] original path (without prefix)
+    # @return [String]
+    def prepend_rootprefix(path)
+      storage.prepend_rootprefix(path)
```

Comment:
I have to admit I didn't specifically tested this one. I just assumed the same problem than `rootprefix=`. Let me check further.